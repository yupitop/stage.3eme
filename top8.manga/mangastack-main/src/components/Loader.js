import { CircularProgress, Backdrop } from '@material-ui/core';
import { makeStyles } from '@material-ui/core/styles';

const useStyles = makeStyles((theme) => ({
  backdrop: {
    zIndex: theme.zIndex.drawer + 1,
    color: '#ff0000',
  },
}));

const Loader = () => {
  const classes = useStyles();
  return (
    <Backdrop className={classes.backdrop} open={true}>
      <CircularProgress size={100} thickness={5} />
    </Backdrop>
  );
};

export default Loader;