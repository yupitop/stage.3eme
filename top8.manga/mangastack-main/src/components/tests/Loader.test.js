import { render } from '@testing-library/react';
import Loader from '../Loader';

it('should render', () => {
  const container = render(<Loader />).container;
  expect(container).toMatchSnapshot();
});