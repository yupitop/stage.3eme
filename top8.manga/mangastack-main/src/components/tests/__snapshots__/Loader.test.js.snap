// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<div>
  <div
    aria-hidden="true"
    class="MuiBackdrop-root makeStyles-backdrop-1"
    style="opacity: 1; webkit-transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms; transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
  >
    <div
      class="MuiCircularProgress-root MuiCircularProgress-colorPrimary MuiCircularProgress-indeterminate"
      role="progressbar"
      style="width: 100px; height: 100px;"
    >
      <svg
        class="MuiCircularProgress-svg"
        viewBox="22 22 44 44"
      >
        <circle
          class="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate"
          cx="44"
          cy="44"
          fill="none"
          r="19.5"
          stroke-width="5"
        />
      </svg>
    </div>
  </div>
</div>
`;
